torch_xla/csrc/xla_op_builder.cpp

```diff
@@ -690,6 +690,7 @@ xla::XlaOp GetDimensionSize(const BuilderPtr& builder,
 xla::XlaOp SetDimensionSize(const BuilderPtr& builder,
                             const std::vector<OpPtr>& operands, py::dict args) {
   int64_t dimension = args["dimension"].cast<int64_t>();
+  // shouldn't `dimension` be a vector<bool> instead of int64_t?
```

Comment:
I grep the XLA reference:
https://www.tensorflow.org/xla/operation_semantics#getdimensionsize

Reply:
`dimension` is the dimension that we want to modified, hence should be a int64